src/site/styles/layout.less

```diff
@@ -93,11 +93,46 @@ input {
     position: relative;
 }
 
-.action-confirmation {
-    position: absolute;
-    top: 0;
-    left: 0;
-    background: white;
+.action-confirmation { 
+    .action-confirmation-overlay {
+        z-index: 200;
+        background: black;
+        position: fixed;
+        opacity: .5;
+        width: 100%;
+        height: 100%;
+        bottom: 0;
+        left: 0;
+        top: 0;
+        right: 0;
+    }
+
+    .action-confirmation-container {
```

Comment:
Migrating comment:
> You're doing an awful lot of width and height setting. Try `display: flex` instead. It should let you specify just the size of the children, and have the container adjust to them.

Migrating response: 
> Layout changes like that should be handled in a wholesale style PR instead of this sort of PR.

I'm confused by what you mean. You're adding the code here, and the comment is on the new code.